### PR TITLE
first commit, skeleton based on nupic.research

### PR DESCRIPTION
**Pull request by lucasosouza**
Just an initial skeleton to get things started. I will continue building upon this today and in the following days.

On requirements, I left requirements-dev the same, but cleaned up requirements a little to avoid conflict with other libraries. This will be heavily modified in the near future. 

Also reused Readme from nupic.research with just a few words change to highlight this is the Embodied repo. 

I modified config.yml but left it commented for now, will uncomment in the following PRs once we migrate the code. 

Tested installation, is working ok after the latest commit with the fixes.

### CODE REVIEW

**Comment by lucasosouza** [comment]
It is failing installation with following error: `pkg_resources.UnknownExtra: ray 0.8.3 has no such extra feature 'tune'`. Maybe this can be happening in nupic.research as well?

**Comment by lucasosouza** [comment]
Another requirements error that is probably also ocurring in nupic.research: `urllib3 1.25.11 is installed but urllib3<1.25,>=1.21.1 is required by {'requests'}`. urllib is set 1.25.11 whereas request has no specified version.

**Review by lscheinkman — CHANGES_REQUESTED** [review]
The installation is failing on a clean environment.  
`conda env create`

**Comment by lscheinkman** [comment]
> Another requirements error that is probably also ocurring in nupic.research: `urllib3 1.25.11 is installed but urllib3<1.25,>=1.21.1 is required by {'requests'}`. urllib is set to 1.25.11 whereas request has no specified version.

**Comment by lucasosouza** [comment]
> The installation is failing on a clean environment.
> Steps to reproduce:
> 
> * `conda env create`

Good catch. I haven't tested env creation, I usually only install from requirements. It is the python version - with 3.7 the two issues above don't exist, it is only when upgrading python. I downgraded to 3.7 for now and will fix it later once we settle on the requirements.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+#       # Cache git-lfs files
+#       - run: git lfs install
+#       - run: git lfs ls-files -l | cut -d' ' -f1 | sort > git-lfs.sha
+#       - restore_cache:
+#           key: git-lfs-files-v1-key-{{ checksum "git-lfs.sha" }}
+#       - run: git lfs pull
+#       - save_cache:
+#           key: git-lfs-files-v1-key-{{ checksum "git-lfs.sha" }}
+#           paths:
+#             -  ~/nta/nupic.embodied/.git/lfs
```
Comment (lscheinkman) [inline .circleci/config.yml:59]: This is only needed if you enable LFS on your repository

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1 @@
+*.pt filter=lfs diff=lfs merge=lfs -text
```
Comment (lscheinkman) [inline .gitattributes:1]: This is only needed if you enable LFS on your repository

File: docs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+source_path = os.path.abspath("../..")
+sys.path.insert(0, source_path)
+
+# Get version from package
+import nupic.embodied  # noqa: E402 isort:skip
+
+# -- Project information -----------------------------------------------------
+
+project = "nupic.embodied"
+copyright = "2019, Numenta"  # noqa: A001
```
Comment (lscheinkman) [inline docs/source/conf.py:47]: 2021

**Review by lscheinkman — APPROVED** [review]
Just small comment regarding LFS

**Pull request merged**